documentation/guide_to_branches_and_PRs.md

```diff
@@ -0,0 +1,71 @@
+# Step-by-Step Guide to a Good Branch and Pull Request (PR)
+
+## Creating a New Branch
+
+1. **Always start from the `main` branch.**
+   - Before starting any new task, ensure you are on the latest `main` branch. 
+   - Pull the latest changes to make sure your branch is up to date, preventing conflicts and ensuring you're working with the current version of the codebase.
+
+2. **Create a new branch for each specific task.**
+   - In the Source Control panel, create a new branch with a clear and descriptive name. 
+   - Use a consistent naming convention such as `feature/short-description` or `bugfix/issue-number` to reflect the task you’re working on.
```

Comment:
Consider also referencing the python style guide (md&d_python_style_guide.md – which for some reason I can only now find in the VIEWS_FAO_index repo). Adding a clear link to discussions on naming conventions may be useful.

Reply:
Make sense - will do

Reply:
And I also thought moved the style guide here - but I can't find it either...

Reply:
It was here at some point I think, but it still exists elsewhere so will be easy to copy over